### PR TITLE
DOCS: Tidies up documentation

### PR DESCRIPTION
**Pull request by cadihowley**
Multiple changes to tidy up documentation:
- Removes all links to the datafiles page on the wiki. This should save a lot of confusion! The docs now reference the fact that required file information is kept in the individual game wiki entry.
- Removes the bulk of the README, and replaces it with an About section, as well as links to the new docs, wiki, changelog, and credits. 
- Adds the Nintendo DS page link to the home page, on the “Handheld consoles” card
- Changes the version displayed under the logo. Instead of “latest” it will display “development build”, and instead of “stable” it will display whichever version of documentation you’re looking at. For example, “v2.2”.
- Tidies up the Android controls section (to comply with style guide), and adds some explanatory text around the three finger tap.
- Changes the way the documentation links between the Settings pages and the config file page for the ```iconspath``` configuration key, namely ```gameid``` no longer linking to the ID field in the GUI settings. I’ve also changed the wording of the description - @antoniou79 could you have a look and tell me if you’re happy with it?
- The tables should finally work properly for ALL screen sizes, and all levels of zoom (including when the sidebar collapses!). Smaller screens will show the columns one below the other (see below), where larger screens will show them as the normal tables. This *SHOULD* make it easier to read, without engaging in too much word wrapping....

Small screen
Large screen

### CODE REVIEW

**Comment by ccawley2011** [comment]
Regarding the README file, can I suggest keeping the first two sections (the equivalent to what's currently included in `docs/QUICKSTART`), so that it still contains a basic overview of how to start using ScummVM and where to report bugs, while referring to the documentation portal and the wiki for more detail?

**Comment by cadihowley** [comment]
@ccawley2011 That sounds very reasonable - I'll make some changes

**Comment by cadihowley** [comment]
I've added a Taskbar integrations page, as per comments on #2926. @criezy if you wouldn't mind casting your eye over that to make sure I haven't missed anything!

**Comment by cadihowley** [comment]
@SupSuper I've made changes to both the settings/graphics page, and the advanced_topics/understand_graphics page. Would you mind having a look to check my explanations?

File: doc/docportal/use_scummvm/taskbar_integration.rst [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+Place the icons in a folder, and either change the *iconspath* in the :doc:`configuration file <../advanced_topics/configuration_file>` to point to this folder, or add the folder to the :ref:`Extra Path <extra>` on the Paths settings tab.
+
+The icon files must be ICO files for Windows, or PNG files for macOS. They must follow one of these naming conventions, where xxx is the file extension:
+
+		- gameid.xxx
+		- :ref:`ID <id>`.xxx
+		- engineid-gameid.xxx
+
+To find the gameid and the engineid, open the :doc:`configuration file <../advanced_topics/configuration_file>`. The gameid is the name of the game in square brackets. The game entry will be similar to this:
```
Comment (SupSuper) [inline doc/docportal/use_scummvm/taskbar_integration.rst:27]: This is incorrect. Both "gameid" and "engineid" are listed under the entry. The name in square brackets [] is the (target) ID, which may or may not match the gameid.

**Comment by SupSuper** [comment]
@cadihowley Looks good to me, had a quick look and commented on some other stuff I noticed.

File: doc/docportal/advanced_topics/configuration_file.rst [hunk 2]
```diff
@@ -208,7 +209,7 @@ There are many recognized configuration keys. In the table below, each key is ei
 		":ref:`frames_per_secondfl <fpsfl>`",boolean,false,
 		:ref:`frontpanel_touchpad_mode <frontpanel>`,boolean, false
 		":ref:`fullscreen <fullscreen>`",boolean,false,
-		":ref:`gameid <gameid>`",string,,
+		gameid,string,,Short name of the game
```
Comment (sev-) [inline doc/docportal/advanced_topics/configuration_file.rst:212]: I would add: "...as used internally, do not change manually"

File: doc/docportal/advanced_topics/understand_graphics.rst [hunk 3]
```diff
@@ -164,4 +186,31 @@ When enabled, ScummVM uses bilinear interpolation instead of nearest neighbor fo
...
-To toggle between bilinear interpolation and nearest neighbor, press :kbd:`Ctrl+Alt+f`.
\ No newline at end of file
+To toggle between bilinear interpolation and nearest neighbor, press :kbd:`Ctrl+Alt+f`.
+
+3D Graphics settings
+---------------------
+
+Vsync
+*******
+V-sync, or vertical sync, synchronizes the frame rate of the game with the monitor's refresh rate. This helps to prevent screen "tearing", where the screen displays parts of multiple frames at once, which looks like a horizontal line where the image appears to split.
```
Comment (sev-) [inline doc/docportal/advanced_topics/understand_graphics.rst:196]: ..with monitor's or video capture refresh rate.

The latter is more important than the former in the modern world: we have almost no CRTs but Twitch.

File: doc/docportal/settings/game.rst [hunk 4]
```diff
@@ -16,12 +16,11 @@ All settings can also be changed in the :doc:`../advanced_topics/configuration_f
 
 ,,,,,,,
 
-.. _gameid:
+.. _id:
 
 **ID**
 	Shows the short name of the game. This ID can be used for launching the game from the command line.
```
Comment (sev-) [inline doc/docportal/settings/game.rst:22]: please add: "You can change it at your convenience, but only letters, numbers and dashes (-) are allowed"

File: doc/docportal/settings/graphics.rst [hunk 5]
```diff
@@ -65,4 +65,27 @@ Filter graphics
...
 
+V-Sync in 3D games
+	If ticked, synchronizes the frame rate of a game with the monitor's refresh rate to prevent screen tearing.
+
+	*vsync*
+
+.. _renderer:
+
+Game 3D renderer
+	Changes how a 3D game is rendered.
```
Comment (sev-) [inline doc/docportal/settings/graphics.rst:78]: Please add: "Has no effect on 2D games"

File: doc/docportal/use_scummvm/taskbar_integration.rst [hunk 6]
```diff
@@ -0,0 +1,46 @@
+===========================
+Taskbar integration
+===========================
+
+The taskbar integration has several features:
+
+- Adds an overlay icon in the taskbar/dock when running a game (Windows and macOS)
```
Comment (sev-) [inline doc/docportal/use_scummvm/taskbar_integration.rst:7]: ...or Ubuntu

**Review by sev- — APPROVED** [review]
Good improvements, I added a few notes and additions.

File: doc/docportal/use_scummvm/taskbar_integration.rst (hunk 6 above)
Comment (cadihowley) [reply doc/docportal/use_scummvm/taskbar_integration.rst:7]: Hmm ok - @criezy suggested it only works on Windows/macOS, which is why it's in there like that. I actually can't get it working on Ubuntu

File: doc/docportal/use_scummvm/taskbar_integration.rst (hunk 1 above)
Comment (cadihowley) [reply doc/docportal/use_scummvm/taskbar_integration.rst:27]: Thank you - fixed.

File: doc/docportal/use_scummvm/taskbar_integration.rst (hunk 6 above)
Comment (criezy) [reply doc/docportal/use_scummvm/taskbar_integration.rst:7]: Indeed the overlay icon is only implemented on Windows and macOS. The taskbar integration on Linux is limited to the progress and count overlays used for the mass add feature.

File: doc/docportal/use_scummvm/taskbar_integration.rst [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+Icons for many games can be found on our `Github repository <https://github.com/scummvm/scummvm-icons>`__. 
+
+Place the icons in a folder, and either change the *iconspath* in the :doc:`configuration file <../advanced_topics/configuration_file>` to point to this folder, or add the folder to the :ref:`Extra Path <extra>` on the Paths settings tab.
+
+The icon files must be ICO files for Windows, or PNG files for macOS. They must follow one of these naming conventions, where xxx is the file extension:
+
+        - :ref:`ID <id>`.xxx
+		- gameid.xxx
+		- engineid-gameid.xxx
+
```
Comment (criezy) [inline doc/docportal/use_scummvm/taskbar_integration.rst:26]: I have just merged a pull request that adds the possibility to use `engineid.xxx`

**Comment by criezy** [comment]
> I've added a Taskbar integrations page, as per comments on #2926. @criezy if you wouldn't mind casting your eye over that to make sure I haven't missed anything!

That looks good to me.

File: doc/docportal/use_scummvm/taskbar_integration.rst (hunk 7 above)
Comment (cadihowley) [reply doc/docportal/use_scummvm/taskbar_integration.rst:26]: Perfect, I'll add that now!

**Pull request merged**